pom.xml

```diff
@@ -47,23 +47,19 @@
     <avro.docDir>build/avro-doc-${project.version}/api</avro.docDir>
 
     <!-- plugin versions -->
-    <apache-rat-plugin.version>0.16.1</apache-rat-plugin.version>
```

Comment:
Are these renames required? People might be overriding specific properties, and now we change them :)

Reply:
I don't think renaming properties is required. I would rather remove the properties (when an override is not needed) to use the depManagement/pluginManagement from the parent.